modules/sdk-coin-arbeth/src/arbeth.ts

```diff
@@ -0,0 +1,29 @@
+import { BaseCoin, BitGoBase, MPCAlgorithm } from '@bitgo/sdk-core';
+import { BaseCoin as StaticsBaseCoin, coins } from '@bitgo/statics';
+import { AbstractEthLikeCoin } from '@bitgo/abstract-eth';
+import { TransactionBuilder as EthTransactionBuilder } from '@bitgo/sdk-coin-eth';
+import { TransactionBuilder } from './lib';
+
+export class Arbeth extends AbstractEthLikeCoin {
+  protected constructor(bitgo: BitGoBase, staticsCoin?: Readonly<StaticsBaseCoin>) {
+    super(bitgo, staticsCoin);
+  }
+
+  static createInstance(bitgo: BitGoBase, staticsCoin?: Readonly<StaticsBaseCoin>): BaseCoin {
+    return new Arbeth(bitgo, staticsCoin);
+  }
+
+  protected getTransactionBuilder(): EthTransactionBuilder {
+    return new TransactionBuilder(coins.get(this.getBaseChain()));
+  }
+
+  /** @inheritDoc */
+  supportsTss(): boolean {
+    return true;
+  }
+
+  /** @inheritDoc */
+  getMPCAlgorithm(): MPCAlgorithm {
+    return 'ecdsa';
+  }
```

Comment:
I suppose this should be default for all eth chains

Reply:
I will add this in the new class that we discussed about abstractEthLikeTssCoin because currently abstractEthLikeCoin class doesn't have it and I think it won't make sense to add it there